imports/plugins/core/shop/server/resolvers/Query/primaryShop.js

```diff
@@ -1,10 +1,10 @@
 /**
- * @name "Query.primaryShop"
+ * @name Query/primaryShop
  * @method
  * @memberof Shop/GraphQL
  * @summary Gets the primary shop
- * @param {Object} parentObject - unused
- * @param {Object} args - unused
+ * @param {Object} _ - unused
```

Comment:
Hi, curious what's the meaning of this change (and what does having "_" mean).

Reply:
@bt3gl we have all our queries / mutations set up to accept these three params: `parentResult`, `inputArgs`, and `context`. In a lot of cases, we don't need to pass in `parentResult` or any `args`, so we use `_` and `__` as placeholders for these fields.